### PR TITLE
Fill icons for restaurant side

### PR DESCRIPTION
**Pull request by mannionaco**
placing these in for restaurant consumption.

### CODE REVIEW

**Comment by lastquestion** [comment]
Now that we have the ability to target subpaths and fill their colors differently, we probably don't need these filled versions anymore. We can probably also remove the other filled icons (there are two I think).

Possibly we need to extend the styleguide to show the filled states available, which I think is what we should do next instead.

We should get the restaurant folks to make sure they can consume the new format and target the paths via CSS, but I don't forsee any problems there.

File: OTKit/otkit-icons/ic_birthday_fill.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
... 16.2238576,8 16.5,8 L17.5,8 C17.7761424,8 18,8.22385763 18,8.5 L18,10 L19,10 C19.5522847,10 20,10.4477153 20,11 L20,15.9000227 L17.1536697,14.1922893 C16.4560307,13.7737217 15.5743664,13.8194647 14.9237891,14.307981 L13.2041277,15.5992693 C12.493296,16.1330306 11.5154755,16.1335695 10.8040558,15.6005921 L9.07581138,14.3058355 C8.42533839,13.8185178 7.54464342,13.7733121 6.84769298,14.1914673 L4,15.9000216 Z M4,17.8991381 L6.84649594,16.1887872 C7.54391042,15.7697374 8.42567171,15.8149443 9.07657369,16.3031208 L10.8,17.5956905 C11.5111111,18.1290239 12.4888889,18.1290239 13.2,17.5956905 L14.9241624,16.3025687 C15.5746951,15.8146692 16.4558699,15.7692125 17.1531539,16.1875829 L20,17.8956905 L20,21 C20,21.5522847 19.5522847,22 19,22 L5,22 C4.44771525,22 4,21.5522847 4,21 L4,17.8991381 Z M11,3.8 C11,3.13706294 12,2 12,2 C12,2 13,3.13706294 13,3.8 C13,4.46293706 12.5525,5 12,5 C11.4475,5 11,4.46293706 11,3.8 Z M6,5.8 C6,5.13706294 7,4 7,4 C7,4 8,5.13706294 8,5.8 C8,6.46293706 7.5525,7 7,7 C6.4475,7 6,6.46293706 6,5.8 Z M16,5.8 C16,5.13706294 17,4 17,4 C17,4 18,5.13706294 18,5.8 C18,6.46293706 17.5525,7 17,7 C16.4475,7 16,6.46293706 16,5.8 Z" id="ic_birthday_Fill" fill="#2D333F"></path>
```
Comment (ot-raypeters) [inline OTKit/otkit-icons/ic_birthday_fill.svg]: Is `id="ic_birthday_Fill"` correct?

File: OTKit/otkit-icons/ic_anniversary_fill.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
....5,2.22385763 17.5,2.5 L17.5,4.5 C17.5,4.74856702 17.3186186,4.95477013 17.0809911,4.99347402 C17.3017452,6.7932474 17.584275,8.12875582 17.9285715,9 C18.1560977,9.57575641 18.4183085,10.1580761 18.7152038,10.7469592 C19.559955,12.4224902 20,14.2727764 20,16.1492115 L20,21.0001314 C20,21.5524161 19.5522847,22.0001314 19,22.0001314 L12.9999941,22.000132 C12.4477093,22.000132 11.9999941,21.5524168 11.9999941,21.000132 L11.9999962,16.1384461 C11.9999974,14.2729494 12.43493,12.4331438 13.270258,10.7651195 C13.5682511,10.1700723 13.8298803,9.58169911 14.0551456,9 C14.3927375,8.12824216 14.6755526,6.79181328 14.903591,4.99071337 Z M16.5547002,13.3698001 C16.2188008,13.1458672 15.7811992,13.1458672 15.4452998,13.3698001 L13.6240377,14.5839749 C13.5691116,14.6205923 13.5219801,14.6677238 13.4853627,14.7226499 C13.3321864,14.9524142 13.3942734,15.2628489 13.6240377,15.4160251 L15.4452998,16.6301999 C15.7811992,16.8541328 16.2188008,16.8541328 16.5547002,16.6301999 L18.3759623,15.4160251 C18.4308884,15.3794077 18.4780199,15.3322762 18.5146373,15.2773501 C18.6678136,15.0475858 18.6057266,14.7371511 18.3759623,14.5839749 L16.5547002,13.3698001 Z" id="ic_anniversary_Fill" fill="#2D333F"></path>
```
Comment (ot-raypeters) [inline OTKit/otkit-icons/ic_anniversary_fill.svg]: Is `id="ic_anniversary_Fill"` correct?
(notice the capital `F`)

Comment (mannionaco) [reply OTKit/otkit-icons/ic_anniversary_fill.svg]: if that doesn't impact dev side it's fine. I can also update and re export easily with a lowercase f

**Comment by ot-raypeters** [comment]
@lastquestion Hey Lin, I want to apologize in advance if I'm jumping the gun here.
 
If we need to undo some of these merges, then no worries -- I am simply trying to unblock this work and access these new icons for CRO.

---

Going forward, I think it would be helpful to have a quick meeting to level-set around expectations and sanity check requirements to validate before merging PRs into `design-tokens`.

What do you think?

**Pull request merged**